include/deal.II/matrix_free/tensor_product_point_kernels.h

```diff
@@ -75,6 +75,32 @@ namespace internal
 
 
 
+  template <int dim, typename Number>
+  inline void
+  compute_values_of_array_in_pairs(
+    dealii::ndarray<Number, 2, dim>                    *shapes,
+    const std::vector<Polynomials::Polynomial<double>> &poly,
+    const Point<dim, Number>                           *p)
+  {
+    const int n_shapes = poly.size();
```

Comment:
Just out of curiosity, why is this `int` and not `unsigned int` or `size_type`? Is this just copy-paste from the existing function, or is there some reason behind it?